### DISCUSSION TITLE
Lesson 7 DECIMALS and INITIAL_PRICE question

### DISCUSSION THREAD

**Discussion by aboronilov** [Cyfrin/foundry-full-course-cu#1607] [Q&A] [answered]
We initiate two constants in file **HelperConfig.s.sol**

``` solidity
    uint8 public constant DECIMALS = 8;
    int256 public constant INITIAL_PRICE = 2000e8;
```

we call that function here 
```solidity
    function getOrCreateAnvilETHConfig() public returns (NetworkConfig memory) {
        if (activeNetworkConfig.priceFeed != address(0)) {
            return activeNetworkConfig;
        }
        // Deploy the mocks
        // Return the mock addesses
        vm.startBroadcast();
        MockV3Aggregator mockPriceFeed = new MockV3Aggregator(
            DECIMALS,
            INITIAL_PRICE
        );
        vm.stopBroadcast();

        NetworkConfig memory anvilConfig = NetworkConfig({
            priceFeed: address(mockPriceFeed)
        });

        return anvilConfig;
    }
```

So how did we decide to set DECIMALS to 8 and INITIAL_PRICE to 2000e8?

**Comment by EngrPips** [comment, marked as answer]
Hello @aboronilov , we were trying to create a mock contract so we just randomly choose the value to work with. do you understand ? 

You could have make them whatever value you like and work with them respectively, considering the actual price feed contract have decimal of `8` so it makes sense that when we are creating a mock contract our decimal is also `8` and as for the price that is more flexible and you can set it to whatever but it is still more sane to mock the  values of the real contract whenever we are creating our mock contract

**Reply by aboronilov** [reply]
Ha cool! And I was thinking ) Thanks a lot

**Reply by EngrPips** [reply]
Anytime buddy, i am glad you figured it out
